Saas/Publish-PerTenantExtensionApps.ps1

```diff
@@ -42,6 +42,8 @@ function Publish-PerTenantExtensionApps {
         $appFiles,
         [ValidateSet('Add','Force')]
         [string] $schemaSyncMode = 'Add',
+        [ValidateSet('','Current version','Next minor version','Next major version')]

```

Comment:
No need for `''`, it's not a mandatory parameter.

Reply:
most pipelines will have code where the value of this comes from a variable, which comes from a file or like. And this:

```
function test {
    Param(
        [ValidateSet('Current version','Next minor version','Next major version')]
        [string] $schedule
    )

    Write-Host "'$test'"
}

$schedule = ''
test -schedule $schedule
```

throws an error:
test : Cannot validate argument on parameter 'schedule'. The argument "" does not belong to the set "Current version,Next minor version,Next major version" specified by the ValidateSet attribute. Supply an 
argument that is in the set and then try the command again.

Which means that people will have to generate two code-paths for this scenario - and we do not want that.